src/AWSS3.jl

```diff
@@ -1259,7 +1282,7 @@ function s3_sign_url(
     end
 end
 
-s3_sign_url(a...; b...) = s3_sign_url(global_aws_config(), a...; b...)
+s3_sign_url(args...; kwargs...) = s3_sign_url(global_aws_config(), args...; kwargs...)
```

Comment:
```suggestion
function s3_sign_url(bucket, path, seconds=3600; kwargs...)
  return s3_sign_url(global_aws_config(), bucket, path, seconds; kwargs...)
end
```

Reply:
no longer valid (followed the below @omus style)